src/pages/filter/common/view-variants/variant-content/variant-content.tsx

```diff
@@ -7,6 +7,7 @@ import { Icon } from '@ui/icon'
 import { Loader } from '@ui/loader'
 import { VariantAspectsLayoutGallery } from '@components/variant-aspects-layout'
 import { TAspectDescriptor } from '@service-providers/dataset-level'
+import { ReturnedVariantsDataCy } from '../../../../../components/data-testid/returned-variants'
```

Comment:
the same {...} from '@components/data-testid...'

Reply:
Fixed using @AlMaQntr suggestion